BigBaseV2/src/hooks/protections/received_clone_sync.cpp

```diff
@@ -25,38 +29,77 @@ namespace big {
 		break;
 	*/
 	
-
+	CBaseModelInfo* get_model_data(rage::joaat_t hash) {
+		auto modelTble = g_pointers->m_model_table;
```

Comment:
Use `const`, this variable isn't modified after being assigned.

Reply:
Forgot, fixed